src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "\\b[A-Z0-9._%-]+@[A-Z0-9.-]+\\.[A-Z]{2,4}\\b";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        if (email == null) {
+            return false;
+        }
+        Pattern pattern = Pattern.compile(EMAIL_PATTERN);
+        Matcher matcher = pattern.matcher(email);
+        return matcher.matches();
```

Comment:
```suggestion
        return email.matches(EMAIL_PATTERN);
```

we can make it simpler